test/server/cards/03_TWI/units/WolfPackEscort.spec.ts

```diff
@@ -0,0 +1,39 @@
+describe('Wolf Pack Escort', function () {
+    integration(function (contextRef) {
+        describe('ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['wolf-pack-escort'],
+                        groundArena: ['duchesss-champion', 'specforce-soldier'],
+                        spaceArena: ['star-wing-scout'],
+                    },
+                    autoSingleTarget: true
+                });
+            });
+
+            it('should return card to hand', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.wolfPackEscort);
+
+                // Checking we cannot select a vehicle
+                expect(context.player1).toBeAbleToSelectExactly([context.duchesssChampion, context.specforceSoldier]);
+
+                // Returning card to hand and checking it was returned
+                context.player1.clickCard(context.specforceSoldier);
+                expect(context.specforceSoldier).toBeInZone('hand');
+            });
+
+            it('can pass ability', function () {
```

Comment:
We don't do explicit tests that the pass ability button works, it's enough to just confirm that it's there. So let's not have this test and instead just have the pass button check as noted in the comment above